### PR TITLE
Update SimpleConsumerExample.java

### PR DESCRIPTION
**Pull request by asnir**
Bug fix: args number should be 5.

### CODE REVIEW

File: kafka-clients/consumer/src/main/java/io/confluent/examples/consumer/SimpleConsumerExample.java [hunk 1]
```diff
@@ -200,12 +200,12 @@ public void run(long maxReads, String topic, int partition, List<String> seedBro
...
   public static void main(String args[]) {
-    if (args.length != 4) {
-      System.out.println("Please provide command line arguments: maxReads partitionId broker port");
+    if (args.length != 5) {
+      System.out.println("Please provide command line arguments: maxReads topic partitionId broker port");
       System.exit(-1);
     }
 
-    long maxReads = Long.parseLong(args[0]);
+    long maxReads = Long.parseLong(args[0]);
```
Comment (miguno) [inline kafka-clients/consumer/src/main/java/io/confluent/examples/consumer/SimpleConsumerExample.java]: Nitpick: unnecessary trailing whitespace (if you remove/revert it, then this line will also not be marked as having changed, which is what we need).

**Comment by miguno** [comment]
Thanks for reporting and the fix, @asnir!  As soon as you have signed the CLA (Contributor License Agreement) -- sorry for that legal inconvenience -- we can merge the updated PR.

**Comment by asnir** [comment]
@miguno where can i find the CLA ?

**Comment by miguno** [comment]
@asnir:  CLA is available at http://clabot.confluent.io/cla.

Normally the CLA bot should have automatically send you instructions via email but apparently it didn't do that (we're looking into this). :-)

**Comment by miguno** [comment]
@asnir: I can't merge your PR unless you sign the CLA (sorry!).  Do you want to take a look again?

**Comment by asnir** [comment]
It doesn't work. I have tried couple of times. I got the following error
"Please try again, something went wrong"

**Comment by miguno** [comment]
@ewencp / @granders : Are you able to help with the CLA / CLAbot woes?

**Comment by ewencp** [comment]
@miguno Not sure why its sometimes getting into a bad state, but its been restarted and I've verified that we have a record for @asnir signing it.

**Comment by miguno** [comment]
@ewencp : Thanks a lot, Ewen for the logistics, and @asnir for the PR.  Merged. :-)

**Pull request merged**